.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - checkout
       - run:
           name: Install System Dependencies
-          command: sudo apt-get update && sudo apt-get install -y libsndfile1
+          command: sudo apt-get update && sudo apt-get install -y libsndfile1 portaudio19-dev
```

Comment:
We could probably remove this and also libsndfile, @mberz?

Reply:
Yes, libsndfile is required. Portaudio is required as well, since it's required by sounddevice. If we do not include it, we would require two different requirements files, which I'd like to avoid.

Reply:
Maybe I'm missing sth, sound device is not running on cirlce ci, it is just for static notebooks, isn't it?

Reply:
You still need to use sounddevice to execute the notebook locally, which is why I added it to the requirements file.
Simply installing it works on CircleCI, with the exception that portaudio is installed. The only thing that causes problems on CircleCI is trying to create audio streams, as no sound card is physically connected.